src/StickyParallaxHeader.js

```diff
@@ -367,7 +370,7 @@ class StickyParallaxHeader extends Component {
 StickyParallaxHeader.propTypes = {
   background: node,
   backgroundColor: string,
-  backgroundImage: number,
+  backgroundImage: oneOfType([object, number]),
```

Comment:
Also, out of curiousity: why is backgroundImage a number?

Reply:
changed prop type